packages/storybook/stories/va-textarea-uswds.stories.jsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const textareaDocs = getWebComponentDocs('va-textarea');
+
+export default {
+  title: `USWDS/Textarea USWDS`,
+  id: 'uswds/va-textarea',
+  parameters: {
+    componentSubtitle: `va-textarea web component`,
+    docs: {
+      page: () => <StoryDocs data={textareaDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  'name': 'my-input',
+  'label': 'Describe your situation',
+  'enable-analytics': false,
+  'required': false,
+  'error': undefined,
+  'maxlength': undefined,
+  'value': undefined,
+  'placeholder': '',
+  'uswds': true,
+  'hint': null,
+};
+
+const Template = ({
+  name,
+  label,
+  'enable-analytics': enableAnalytics,
+  required,
+  error,
+  maxlength,
+  value,
+  placeholder,
+  uswds,
+  hint,
+}) => {
+  return (
+    <va-textarea
+      uswds={uswds}
+      name={name}
+      label={label}
+      enable-analytics={enableAnalytics}
+      required={required}
+      error={error}
+      hint={hint}
+      maxlength={maxlength}
+      value={value}
+      placeholder={placeholder}
+      onBlur={e => console.log('blur event', e)}
+      onInput={e => console.log('input event value', e.target.value)}
+    />
+  );
+};
+
+const ResizableTemplate = ({
+  name,
+  label,
+  'enable-analytics': enableAnalytics,
+  required,
+  error,
+  maxlength,
+  value,
+  placeholder,
+  uswds,
+  hint,
+}) => {
+  return (
+    <va-textarea
+      uswds={uswds}
+      class="resize-y"
```

Comment:
This resize-y class doesn't seem to be working in the v3 version like it does in v1. I can still resize the v3 version in both directions.